src/t8_geometry/t8_geometry.h

```diff
@@ -39,6 +39,8 @@ typedef enum t8_geometry_type {
   T8_GEOMETRY_TYPE_LINEAR,
   /** The linear, axis aligned geometry uses only 2 vertices, since it is axis aligned. */
   T8_GEOMETRY_TYPE_LINEAR_AXIS_ALIGNED,
+  /** ... . */
```

Comment:
```suggestion
  /** The lagrange geometry uses a lagrange mapping for curved elements. */
```
You can also decide for yourself how to describe your implementations